templates/views/collection.jade

```diff
@@ -30,13 +30,24 @@ block modal
 
 block topLevelPreview
     h3 Members
+    if types
+        form(name="filterTypes", method="post").submission-form
```

Comment:
This form looks squished together. Should be a proper button addon as shown [here](https://getbootstrap.com/docs/3.3/components/#input-groups-buttons).

Reply:
@3ach I'm not sure if I've resolved this one that way you were thinking.  Could you check and see if there was something else you had in mind?